metadata-ingestion/scripts/datahub_preflight.sh

```diff
@@ -45,7 +45,7 @@ arm64_darwin_preflight() {
   	pip3 install --no-use-pep517 scipy
   fi
 
-  brew_install "openssl@1.1"
+  brew_install "openssl@3.0.14"
```

Comment:
Quick question - Why this change?

Reply:
I needed to upgrade the dependency version because the 1.1 version was deprecated.

Reply:
I reverted to the previous version